spec/requests/api/v1/targets/create_spec.rb

```diff
@@ -13,83 +13,97 @@
+          end
         end
-      end
 
-      context 'when radius is incorrect' do
-        let(:params) do
-          { target: attributes_for(:target).merge(topic_id: topic_id, radius: nil) }
-        end
+        context 'when radius is incorrect' do
+          let(:params) do
+            { target: attributes_for(:target).merge(topic_id: topic_id, radius: nil) }
+          end
 
-        it 'returns unprocessable entity' do
-          subject
-          expect(response).to have_http_status(:bad_request)
+          it 'returns bad request entity' do
+            subject
+            expect(response).to have_http_status(:bad_request)
+          end
         end
-      end
 
-      context 'when title is incorrect' do
-        let(:params) do
-          { target: attributes_for(:target).merge(topic_id: topic_id, title: nil) }
-        end
+        context 'when title is incorrect' do
+          let(:params) do
+            { target: attributes_for(:target).merge(topic_id: topic_id, title: nil) }
+          end
 
-        it 'returns unprocessable entity' do
-          expect(response).to have_http_status(:bad_request)
+          it 'returns bad request' do
+            expect(response).to have_http_status(:bad_request)
+          end
         end
-      end
 
-      context 'when longitude is incorrect' do
-        let(:params) do
-          { target: attributes_for(:target).merge(topic_id: topic_id, longitude: nil) }
+        context 'when longitude is incorrect' do
+          let(:params) do
+            { target: attributes_for(:target).merge(topic_id: topic_id, longitude: nil) }
+          end
+
+          it 'returns bad request' do
+            expect(response).to have_http_status(:bad_request)
+          end
         end
 
-        it 'returns unprocessable entity' do
-          expect(response).to have_http_status(:bad_request)
+        context 'when latitude is incorrect' do
+          let(:params) do
+            { target: attributes_for(:target).merge(topic_id: topic_id, latitude: nil) }
+          end
+
+          it 'returns bad request' do
+            expect(response).to have_http_status(:bad_request)
+          end
         end
       end
 
-      context 'when latitude is incorrect' do
-        let(:params) do
-          { target: attributes_for(:target).merge(topic_id: topic_id, latitude: nil) }
-        end
+      context 'when user is not logged in' do
+        let(:headers) { {} }
 
-        it 'returns unprocessable entity' do
-          expect(response).to have_http_status(:bad_request)
+        it 'returns unauthorized' do
+          subject
+          expect(response).to have_http_status(:unauthorized)
         end
       end
     end
 
-    context 'when user is not logged in' do
-      let(:headers) { {} }
+    context 'when have more than 10 targets' do
+      before do
+        allow_any_instance_of(Targets::CreateService)
+          .to receive(:call).and_raise(MaxAllowTargetsError)
+      end
```

Comment:
This test should actually create 10 targets so that the error raises by itself, no need to mock it.